src/shared/seccomp-util.c

```diff
@@ -213,6 +231,84 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "sigreturn\0"
                 "time\0"
         },
+        [SYSCALL_FILTER_SET_FILE_SYSTEM] = {
+                /* Basic file system access (reading file properties, opening/creating files and directories for
+                 * read+write) */
+                .name = "@file-system",
```

Comment:
Hmm, I figure I'll drop the file-system-access bit for now, and split it into a separate PR. I'll leave the basic IO stuff in the patch however.

@keszybz close() already is in the @basic-io group in this patch. And I was afraid of adding fcntl since it's one of those multplixed calls, and we'd blanket allow quite a few ops that way. I sounded better to me that people needing it would explicitly say "SystemCallFilter=@basic-io fcntl" or so...